src/test/java/nl/tudelft/ewi/devhub/webtests/AssignmentReviewTest.java

```diff
@@ -0,0 +1,106 @@
+package nl.tudelft.ewi.devhub.webtests;
+
+import com.google.inject.Inject;
+import nl.tudelft.ewi.devhub.server.database.controllers.CourseEditions;
+import nl.tudelft.ewi.devhub.server.database.controllers.Deliveries;
+import nl.tudelft.ewi.devhub.server.database.controllers.Groups;
+import nl.tudelft.ewi.devhub.server.database.entities.Assignment;
+import nl.tudelft.ewi.devhub.server.database.entities.CourseEdition;
+import nl.tudelft.ewi.devhub.server.database.entities.Delivery;
+import nl.tudelft.ewi.devhub.server.database.entities.Group;
+import nl.tudelft.ewi.devhub.webtests.utils.WebTest;
+import nl.tudelft.ewi.devhub.webtests.views.DeliveryReviewView;
+import org.junit.Test;
+
+import java.text.ParseException;
+
+import javax.persistence.EntityManager;
+
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Douwe Koopmans on 9-5-16.
+ */
+public class AssignmentReviewTest extends WebTest{
+    private static final String ASSISTANT_USERNAME = "assistant1";
+    private static final String ASSISTANT_PASSWORD = "assistant1";
+
+    @Inject
+    private CourseEditions courseEditions;
+
+    @Inject
+    private Deliveries deliveries;
+
+    @Inject
+    private EntityManager entityManager;
+
+    @Inject
+    private Groups groups;
+
+    @Test
+    public void testDeliverable() throws ParseException {
+        final DeliveryReviewView view = getDeliveryReviewView();
+
+        final CourseEdition course = courseEditions.find(1);
+        final Group group = groups.find(course).get(0);
+        final Delivery modelDelivery = deliveries.find(group,2L);
+        final Assignment modelAssignment = modelDelivery.getAssignment();
+        final DeliveryReviewView.Assignment viewAssignment = view.getAssignment();
+
+        assertEquals(modelDelivery.getCreatedUser().getName(), viewAssignment.getAuthor());
+        assertEquals(modelAssignment.getName(), viewAssignment.getName());
+        assertEquals(modelDelivery.getState(), viewAssignment.getStatus());
+    }
+
+    @Test
+    public void testReviewForm() throws ParseException {
+        final DeliveryReviewView view = getDeliveryReviewView();
+
+        final DeliveryReviewView.DeliveryForm deliveryForm = view.getDelivery();
+        deliveryForm.setGrade("4.0");
+        deliveryForm.setState("Rejected");
+        deliveryForm.setCommentary("Hello World!");
```

Comment:
We should store this in a static variable so it can be reused in the test (and if we change it, we don't have to change statements elsewhere in the test)